src/RunTests/build/Microsoft.Build.RunTests.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <UsingTask TaskName="Microsoft.Build.RunTestsTask" AssemblyFile="$(MSBuildThisFileDirectory)netstandard2.0\Microsoft.Build.RunTests.dll" />
+  <Target Name="RunTests" AfterTargets="MSBuildRunTests" Condition="$(IsTestProject)==true">
```

Comment:
Would it make sense to name this VSTest after the existing task we have in dotnet sdk? I don't know if this task will replace the on in the dotnet sdk, but if user has existing pipeline in place that is using `/t:VSTest`, we should not break them.

`<Target Name="VSTest" DependsOnTargets="ShowInfoMessageIfProjectHasNoIsTestProjectProperty">`

Reply:
I think that makes sense, especially if we're reasonable sure it won't break any existing pipelines